examples/basic/index.html

```diff
@@ -1,7 +1,10 @@
-<!doctype html public "embarassment">
+<!doctype html>
+<html>
+<head>
 <title>Basic Example</title>
 <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css" rel="stylesheet">
 <link href="../bootstrap-datetimepicker.min.css" rel="stylesheet">
```

Comment:
I'm not sure what you are referring to being removed? If you mean the style sheet it was replaced with the current one from eonasdan-bootstrap-datetimepicker in commit [7aca80be75649f4987f0b62b61f61646ef63dd34](https://github.com/unthunk/react-bootstrap-datetimepicker/commit/7aca80be75649f4987f0b62b61f61646ef63dd34).